pages/index.tsx

```diff
@@ -1,8 +1,34 @@
-import { Vote } from "components";
+import {
+  Banner,
+  HowItWorks,
+  Layout,
+  PageInnerWrapper,
+  PageOuterWrapper,
+  Votes,
+} from "components";
+import { Strong } from "components/pages/styles";
+import { truncateDecimals } from "helpers";
+import { useGlobals } from "hooks";
 import type { NextPage } from "next";
 
 const VotePage: NextPage = () => {
-  return <Vote />;
+  const {
```

Comment:
Having this redundant component within a component structure is unnecessary. I will update the other pages to work like this in another PR.